service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/DocumentMetaData.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.fpl.model.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(force = true)
+public class DocumentMetaData {
+    protected final DocumentReference typeOfDocument;
+    protected LocalDateTime dateTimeUploaded;
+    protected String uploadedBy;
```

Comment:
why protected?

Reply:
to use Builder on subClasses, I can't have it private.